parsec/core/gui/instance_widget.py

```diff
@@ -47,6 +47,24 @@ async def _do_run_core(config, device, qt_on_ready):
                 await trio.sleep_forever()
 
 
+def check_macfuse_version() -> bool:
+    import os
+
+    macfuse_plist_path = "/Library/Filesystems/macfuse.fs/Contents/Info.plist"
+    if not os.path.exists(macfuse_plist_path):
+        # osxfuse installed, not macfuse
+        return False
+
+    import plistlib
+    import requests
+
+    local_version = plistlib.readPlist(macfuse_plist_path)["CFBundleVersion"]
+    remote_version = requests.get("https://api.github.com/repos/osxfuse/osxfuse/releases/latest")
+    if local_version < remote_version.json()["name"].split()[1]:
```

Comment:
So we ask the user to update his version of macfuse every time a new release occurs upstream ?

Shouldn't we just set a minimal macfuse version (that we update each time we do a new Parsec release) and compare to this one ? this would have to benefits:
- no need for a http request ;-)
- less annoyance to the user considering how new releases [are rather abundant](https://github.com/osxfuse/osxfuse/releases), besides it should be more acceptable for the user to update his macfuse version when he choose to update Parsec rather than at any random time